content/master/concepts/managed-resources.md

```diff
@@ -635,21 +635,143 @@ my-rds-instance      True    True     my-custom-name       11m
 
 ### Creation annotations
 
-Providers create new managed resources with the
-`crossplane.io/external-create-pending` annotation.
+When an external system like AWS generates nondeterministic resource names it's
+possible for a provider to create a resource but not record that it did. When
+this happens the provider can't manage the resource.
 
-The Provider applies the `crossplane.io/external-create-succeeded` or
-`crossplane.io/external-create-failed` annotation after making the external API
-call and receiving a response. 
+{{<hint "tip">}}
+Crossplane calls resources that a provider creates but doesn't manage _leaked
```

Comment:
```suggestion
Crossplane calls resources that a provider created but lost track of and therefore doesn't manage _leaked
```

Reply:
trying to word in such a way so that readers would understand instantly that this is an error case, without using the word "error", "exception" or "rare"